larq_compute_engine/tflite/kernels/bconv2d.cc

```diff
@@ -395,9 +393,9 @@ void OneTimeSetup(TfLiteContext* context, TfLiteNode* node, OpData* op_data) {
 
 // Fill in the OutputTransform values for float and/or int8 outputs
 template <typename DstScalar>
-void GetOutputTransform(OutputTransform<DstScalar>& output_transform,
-                        TfLiteContext* context, TfLiteNode* node,
-                        OpData* op_data) {
+inline void GetOutputTransform(OutputTransform<DstScalar>& output_transform,
```

Comment:
Is this `inline` added here for speed? If so, I think you'll need to use something more aggressive if you really want the compiler to do inlining, e.g.:
```c++
#if defined(__GNUC__)
#define FORCE_INLINE __attribute__((always_inline)) inline
#else
// Optionally add more force-inlines for other compilers here, this is the fallback:
#define FORCE_INLINE inline
#endif
```

Reply:
Yeah I guess it's more that all the other utility function like this had `inline` so I added it here too. I don't think the speed matters much here - it will get called once per op per `Eval` - so probably not worth doing the force inline.

Reply:
For what its worth: this `inline` keyword deals mostly with the one-definition-rule (ODR) of C/C++ object files. The linker will complain if two `.cc` files both define the same function, except if its marked `inline`, this tells the linker: "hey i manually made sure these two functions contain exactly the same contents, so just pick any of the two".
This is very useful if you define functions in `.h` files that get included by multiple `.cc` files.
For functions defined in `.cc` files like this one, it is actually not needed, and one could argue that its better to remove `inline` so that we *do* get a warning in case another `.cc` file defines a function with the same name and signature (but not necessarily the same content).

I'm not sure how much the keyword affects *actual* inlining of the function.